administration/live-migration.adoc

```diff
@@ -32,6 +32,8 @@ data:
   feature-gates: "DataVolumes,LiveMigration"
 ....
 
+Additionally, Virtual machines using a PersistentVolumeClaim (PVC) must have a shared ReadWriteMany (RWX) access mode to be live migrated.
```

Comment:
There is a similar comment on line 61, in "Condition and migration method"

Reply:
Should we move that one to the upper part to make it clear that is a requirement and not just a 'rejection' ?

Reply:
Perhaps we need a new section with "Limitations", as we also won't allow a migration with a bridge interface attached to a pod network...[1]

https://kubevirt.io/user-guide/docs/latest/creating-virtual-machines/interfaces-and-networks.html

Reply:
There is also another PR[1] in progress that will add new limitations.

[1] https://github.com/kubevirt/kubevirt/pull/2982

Reply:
I'll add mention to it or should we have that other 'on watch' to update when merged?

Reply:
I'll add a comment in the PR ^^ to update this section after it's merged.

Reply:
Done!